src/components/Tabs/ScrollableTabs/ScrollableTabs.st.css

```diff
@@ -0,0 +1,65 @@
+:import {
+    -st-from: "../../../tpaStyles/styleableExt.js";
```

Comment:
is the .js needed (not according to the Stylable docs)
https://stylable.io/docs/references/formatters